app/controllers/openid_application_controller.rb

```diff
@@ -11,6 +11,11 @@
 class OpenidApplicationController < ApplicationController
   before_action :authenticate
   TOKEN_REGEX = /Bearer /.freeze
+  class ValidationError < StandardError
```

Comment:
Wondering if there is a better place to define custom error types?

Reply:
I think I would prefer making a series of Error classes under Common::Exceptions that deal specifically with OAuth, since this isn't the only place tokens are used. This would make it easier to identify/handle in the Application Controller as well. However, I am unsure if the health team would find this useful or use it at all. I would love to hear your thoughts on this idea.

Reply:
@edmkitty If you're talking about the team that manages [this health api](https://developer.va.gov/explore/health/docs/fhir?version=current), then they live outside of the context of vets-api. Kong forwards to a set of Java applications with their own client-facing error handling.

Reply:
At the moment it looks like only `veteran_verification` and `claims` are using the `OpenidApplicationController` in vets-api.